discovery-infra/day2.py

```diff
@@ -25,13 +32,18 @@ def execute_day2_flow(ocp_cluster_id, args):
     ocp_openshift_version = ocp_cluster.openshift_version
     ocp_api_vip_dnsname = "api." + ocp_cluster_name + "." + ocp_cluster.base_dns_domain
     ocp_api_vip_ip = ocp_cluster.api_vip
-    cluster_id = str(uuid.uuid4())
-    cluster = client.create_day2_cluster(
-        ocp_cluster_name + "-day2", cluster_id, **_day2_cluster_create_params(ocp_openshift_version, ocp_api_vip_dnsname)
-    )
-
-    set_cluster_pull_secret(client, cluster_id, args.pull_secret)
+    if day2_type_flag == "ocp":
+        cluster = ocp_cluster
+        terraform_cluster_dir_prefix = "test-infra-cluster-assisted-installer"
+    else:
+        cluster_id = str(uuid.uuid4())
+        cluster = client.create_day2_cluster(
+            ocp_cluster_name + "-day2", cluster_id, **_day2_cluster_create_params(ocp_openshift_version, ocp_api_vip_dnsname)
+        )
+        set_cluster_pull_secret(client, cluster_id, args.pull_secret)
+        terraform_cluster_dir_prefix = ocp_cluster_name
```

Comment:
i think after adding new  ocp type name as ocp_cluster_name is a little bit confusing if it is not unders if ... == ocp

Reply:
not sure i understand your comment